proctor-consumer/src/main/java/com/indeed/proctor/consumer/AbstractGroupsManager.java

```diff
@@ -83,27 +96,57 @@ protected ProctorResult determineBucketsInternal(final Identifiers identifiers,
                     emptyMap()
             );
         }
-        final ProctorResult proctorResult = proctor.determineTestGroups(identifiers, context, forcedGroups);
+        final ProctorResult proctorResult = proctor.determineTestGroups(
+                identifiers,
+                context,
+                forceGroupsOptions,
+                emptyList()
+        );
 
         interceptor.afterDetermineGroups(proctorResult);
         return proctorResult;
     }
 
+    /**
+     * @deprecated Use {@link #determineBucketsInternal(HttpServletRequest, HttpServletResponse, Identifiers, Map, boolean)}
```

Comment:
The alternative method should be the one without request or response.

Reply:
Good catch. I clarified intention of this message.